app/models/subtransaction_payment.rb

```diff
@@ -13,7 +13,9 @@ class SubtransactionPayment < ApplicationRecord
 	has_one :supporter, through: :subtransaction
 	has_one :nonprofit, through: :subtransaction
 
-	delegated_type :paymentable, types: %w[OfflineTransactionCharge OfflineTransactionDispute OfflineTransactionRefund StripeCharge StripeRefund]
+	delegated_type :paymentable,
+																types: %w[OfflineTransactionCharge OfflineTransactionDispute OfflineTransactionRefund StripeCharge
+																										StripeRefund]
```

Comment:
Try to split the types onto separate lines so we don't end up with those weird spacing issues